README.md

```diff
@@ -336,6 +344,11 @@ The following options are supported:
 
 - `password`: password for the network (required if `wpa-psk` or `sae` is used)
 
+#### `type: dummy`
```

Comment:
Is `dummy` term used by nm? Is it possible to replace it with terms suggested by woke?

Reply:
`dummy` term used by nm.  I prefer to stick to it, and put the HTML comment `<!--- wokeignore:rule=dummy --->` in the readme seems not to be a nice solution.